astropy/time/core.py

```diff
@@ -2369,18 +2369,26 @@ def to_value(self, *args, **kwargs):
         To convert to a ``TimeDelta`` format and optional sub-format the options
         are::
 
-          tm = TimeDelta(1.0 * u.s)
-          tm.to_value('jd')  # equivalent of tm.jd
-          tm.to_value('jd', 'decimal')  # convert to 'jd' as a Decimal object
-          tm.to_value('jd', subfmt='decimal')
-          tm.to_value(format='jd', subfmt='decimal')
+          >>> tm = TimeDelta(1.0 * u.s)
+          >>> tm.to_value('jd')  # equivalent of tm.jd
```

Comment:
For floating point comparisons, please use `# doctest: +FLOAT_CMP` provided by [pytest-doctestplus](https://github.com/astropy/pytest-doctestplus) plugin.

Reply:
Another trick is to use `27*u.s` in the example, then one gets a nice fraction of a day.

Reply:
These changes are again *reducing* the clarity of the docstring by introducing visual noise. This is intended as a concise (short) demonstration of the syntax. Detailed examples should be in the narrative docs.

Is it possible to run the examples without checking the output?